### PR TITLE
bat: update Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages.es/common/bat.md [hunk 1]
```diff
@@ -4,26 +4,34 @@
...
+- Imprime el contenido de uno o más archivos en `stdout`:
 
-`bat {{archivo}}`
+`bat {{ruta/al/archivo1 ruta/al/archivo2 ...}}`
 
-- Concatena varios archivos creando un nuevo archivo:
+- Concatena varios archivos en el archivo de destino:
 
-`bat {{archivo1}} {{archivo2}} > {{archivo_final}}`
+`bat {{ruta/al/archivo1 ruta/al/archivo2 ...}} > {{ruta/al/archivo_de_objetivo}}`
```
Comment (ikks) [inline pages.es/common/bat.md:13]: Maybe {{ruta/al/archivo_destino}} ?

File: pages.es/common/bat.md [hunk 2]
```diff
@@ -4,26 +4,34 @@
...
-`bat --language json {{archivo.json}}`
+`bat {{-A|--show-all}} {{ruta/al/archivo}}`
 
-- Muestra todos los lenguajes permitidos:
+- Elimina todos los adornos excepto los números de línea en la salida:
 
-`bat --list-languages`
+`bat {{-n|--number}} {{ruta/al/archivo}}`
+
+- Resalta sintácticamente un archivo JSON estableciendo explícitamente el idioma:
```
Comment (ikks) [inline pages.es/common/bat.md:31]: Better to use lenguaje i.e. json, to not confuse with idioma i.e. es

File: pages.es/common/bat.md [hunk 3]
```diff
@@ -4,26 +4,34 @@
...
+
+`bat {{-l|--language}} json {{ruta/al/archivo.json}}`
+
+- Muestra todos los idiomas soportados:
```
Comment (ikks) [inline pages.es/common/bat.md:35]:
```suggestion
- Muestra todos los lenguajes soportados:
```

File: pages.es/common/bat.md [hunk 4]
```diff
@@ -4,26 +4,34 @@
...
-`bat --language json {{archivo.json}}`
+`bat {{-A|--show-all}} {{ruta/al/archivo}}`
 
-- Muestra todos los lenguajes permitidos:
+- Elimina todos los adornos excepto los números de línea en la salida:
 
-`bat --list-languages`
+`bat {{-n|--number}} {{ruta/al/archivo}}`
+
+- Resalta sintácticamente un archivo JSON estableciendo explícitamente el idioma:
```
Comment (ikks) [inline pages.es/common/bat.md:31]:
```suggestion
- Resalta sintácticamente un archivo JSON estableciendo explícitamente el lenguaje:
```

File: pages.es/common/bat.md [hunk 5]
```diff
@@ -4,26 +4,34 @@
 > Un clon de `cat` con resaltado de sintaxis e integración con Git.
 > Más información: <https://github.com/sharkdp/bat>.
 
-- Imprime los contenidos de un archivo a la salida estándar:
+- Imprime el contenido de uno o más archivos en `stdout`:
```
Comment (ikks) [inline pages.es/common/bat.md:7]:
```suggestion
- Imprime bellamente (pretty print) el contenido de uno o más archivos en `stdout`:
```

File: pages.es/common/bat.md [hunk 6]
```diff
@@ -4,26 +4,34 @@
...
 
-`bat {{archivo}}`
+`bat {{ruta/al/archivo1 ruta/al/archivo2 ...}}`
 
-- Concatena varios archivos creando un nuevo archivo:
+- Concatena varios archivos en el archivo de destino:
```
Comment (ikks) [inline pages.es/common/bat.md:11]:
```suggestion
- Concatena varios archivos en el archivo destino:
```

**Review by ikks — APPROVED** [review]

**Pull request merged**